.changeset/funny-spies-type.md

```diff
@@ -0,0 +1,8 @@
+---
+'@commercetools-uikit/secondary-icon-button': patch
+'@commercetools-uikit/secondary-button': patch
+'@commercetools-uikit/primary-button': patch
+'@commercetools-uikit/design-system': patch
```

Comment:
I think some packages are missing, could you please re-generate the changeset?

Reply:
Good catch, thanks! Re-added again 😄